deno_typescript/lib.rs

```diff
@@ -259,10 +255,6 @@ pub fn get_asset(name: &str) -> Option<&'static str> {
   }
   match name {
     "bundle_loader.js" => Some(include_str!("bundle_loader.js")),
-    "lib.deno_runtime.d.ts" => {
-      Some(include_str!("../cli/js/lib.deno_runtime.d.ts"))
```

Comment:
Note: this is the reason for the revert - this breaks "cargo package".

Reply:
I wonder why the cargo package needs to access this?  I guess it might have to do with supporting `--types` in the package, but I thought we didn't need it here anymore.